builtin/game/privileges.lua

```diff
@@ -28,6 +28,15 @@ function core.register_privilege(name, param)
 	core.registered_privileges[name] = def
 end
 
+function core.override_privilege(name, redefinition)
+	local privilege = core.registered_privileges[name]
+	assert(privilege, "Attempt to override non-existent privilege "..name)
```

Comment:
An `error` call would probably be better here, as it can also output the stack trace-back if the second parameter is set to 2.

```suggestion
	error(privilege, "Attempt to override non-existent privilege " .. name, 2)
```

Reply:
Now fixed. It looks better now. 🙂